Cannot generate a one-to-one relationship with the latest version of jhipster and jhipster-uml

one-to-one entity creation now requires `otherEntityRelationshipName` and `otherEntityField` to be set when `ownerSide == true`

this is since https://github.com/jhipster/generator-jhipster/pull/1757

Sorry fogot to mention this when developing this PR.

One will get the following error trying to generate a one to one relationship with jhipster-uml
```
The entity Child is being created.
WARNING otherEntityRelationshipName is missing in .jhipster/Child.json for relationship with id 1, using child as fallback
ERROR otherEntityField is missing in .jhipster/Child.json for relationship with id 1
```

@andidev When generating an entity with a One-to-One not-owner relationship with java 7 there is this error:
[ERROR] /home/user/Documents/jhipsterproject/src/main/java/com/mycompany/myapp/web/rest/JobResource.java:[82,24] cannot find symbol
[ERROR] symbol:   method getSecond()
[ERROR] location: variable job of type com.mycompany.myapp.domain.Job

This ligne correspond to this ![ one ](https://github.com/jhipster/generator-jhipster/blob/9d67fc06a5ce691512038d89079e94ccdeab4ebc/entity/templates/src/main/java/package/web/rest/_EntityResource.java#L108) on the template _EntityResource.java.

I wonder what the "getSecond()" method is and how comes it hasn't been initialize.

--- a/entity/templates/src/main/java/package/web/rest/_EntityResource.java
+++ b/entity/templates/src/main/java/package/web/rest/_EntityResource.java
@@ -110,7 +110,7 @@ public class <%= entityClass %>Resource {
             log.debug("REST request to get all <%= entityClass %>s where <%= relationships[idx].relationshipName %> is null");<% if (javaVersion == '7') { %>
             List <%= entityInstance %>s = new ArrayList<<%= entityClass %>>();
             for (<%= entityClass %> <%= entityInstance %> : <%= entityInstance %>Repository.findAll()) {
-                if (<%= entityInstance %>.getSecond() == null) {
+                if (<%= entityInstance %>.get<%= relationships[idx].relationshipNameCapitalized %>() == null) {
                     <%= entityInstance %>s.add(<%= entityInstance %>);
                 }
             }

